src/Microsoft.AspNet.Razor/Parser/CSharpCodeParser.Directives.cs

```diff
@@ -287,7 +298,9 @@ private void TagHelperDirective(string keyword, Func<string, ISpanCodeGenerator>
 
             if (EndOfFile || At(CSharpSymbolType.NewLine))
             {
-                Context.OnError(CurrentLocation, RazorResources.FormatParseError_DirectiveMustHaveValue(keyword));
+                Context.OnError(CurrentLocation,
```

Comment:
also not sure here: does `CurrentLocation` still reference the directive's spot?